### DISCUSSION TITLE
CLI and --dialect option documentation

### DISCUSSION THREAD

**Discussion by aborruso** [frictionlessdata/frictionlessdata.io#743] [General]
Hi,
I'm writing a CLI guide in Italian, and I'm starting to documenting `validate` options.

I'm writing some recipes, like "How to set CSV delimiter in validation":

```
frictionless validate --dialect '{"delimiter": ";"}' input.csv
```

What's the best documentation page to read about CSV list of configurations and their meaning I can use via `dialect` option?

I thought the page could be [this one](https://framework.frictionlessdata.io/docs/references/formats-reference#csv), but here I do not have the name of the parameter, I have the meaning and the type.

Can I use the [list of arguments of the API](https://framework.frictionlessdata.io/docs/references/api-reference#csvdialect)?

Thank you

**Comment by roll** [comment]
Hi @aborruso,

I think, currently, the best source will be the initial CSV Dialect spec - https://specs.frictionlessdata.io/csv-dialect/#specification

There will be some changes regarding this in v5 and we will have better documentation when it's released

**Reply by aborruso** [reply]
Thank you

**Comment by rgaiacs** [comment]
@aborruso Can you share the url to the CLI guide in Italian? Thanks!

**Reply by aborruso** [reply]
> @aborruso Can you share the url to the CLI guide in Italian? Thanks!

Sure. It is still in its very early stages. I have written:

- an intro for describe https://arigadicomando.it/frictionless/descrivere/
- an intro for validate https://arigadicomando.it/frictionless/validare/
- some recipes https://arigadicomando.it/frictionless/ricette/

@rgaiacs do you speak Italian?

**Comment by rgaiacs** [comment]
@aborruso Thanks for share the links.

I don't speak Italian. :-/ I speak English and Portuguese. And a little bit of Spanish.

I'm writing a few notes about Frictionless Data, see https://raniere-phd.gitlab.io/frictionless-data-handbook/, and I want to compare notes. :-D

**Reply by aborruso** [reply]
> I'm writing a few notes about Frictionless Data, see https://raniere-phd.gitlab.io/frictionless-data-handbook/, and I want to compare notes. :-D

you are much, much further ahead. Well done

**Reply by roll** [reply]
Great work @rgaiacs! (cc @sapetti9)
